src/Configuration/src/RandomValue/RandomValueProvider.cs

```diff
@@ -197,6 +197,6 @@ private string GetRandomBytes()
     {
         byte[] bytes = new byte[16];
         Random.Shared.NextBytes(bytes);
-        return BitConverter.ToString(bytes).Replace("-", string.Empty, StringComparison.Ordinal);
+        return Convert.ToHexString(bytes);
```

Comment:
This is explained at https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-9, specifically:

![image](https://github.com/user-attachments/assets/77366d0e-141d-47be-b704-a3f6b29c9941)